src/app/components/BibPage/BibMainInfo.jsx

```diff
@@ -1,34 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router';
 import { isEmpty as _isEmpty } from 'underscore';
 
 const BibMainInfo = ({ bib = {} }) => {
   if (_isEmpty(bib)) return null;
 
-  const title = bib.title && bib.title.length ? bib.title[0] : '';
   const materialType = bib && bib.materialType && bib.materialType[0] ?
     bib.materialType[0].prefLabel : null;
+  const author = bib.creatorLiteral && bib.creatorLiteral.length ?
+    `${bib.creatorLiteral[0]}` : '';
+  const contributor = bib.contributorLiteral && bib.contributorLiteral.length ?
+    bib.contributorLiteral : null;
```

Comment:
Hmm yeah, you're using the first name in the array as the main author and then the rest as contributors. I'm not sure how the API or the data is structure or if there's a different field that should be used instead.

I think it's fine for now but it's definitely something to come back to once we have more data in the API (if it's not there already).